action.yml

```diff
@@ -58,8 +59,8 @@ runs:
         aws_region: ${{ inputs.aws_region }}
 
     - name: Render and deploy task definition to Amazon ECS
-      id: deploy-task-definition
-      uses: moneymeets/action-ecs-deploy/custom-deploy-steps/create-task-definition@master
+      id: deploy-production-task-definition
+      uses: moneymeets/action-ecs-deploy/custom-deploy-steps/create-task-definition@feature/MD-7199-handle-ECS-deployment-failures-and-rollback
```

Comment:
Just a reminder for us to change this (and other places) back to master before merging.